helm/PROJECT

```diff
@@ -0,0 +1,20 @@
+# Code generated by tool. DO NOT EDIT.
+# This file is used to track the info used to scaffold your project
+# and allow the plugins properly work.
+# More info: https://book.kubebuilder.io/reference/project-config.html
+domain: quay.io
+layout:
+- helm.sdk.operatorframework.io/v1
+plugins:
+  manifests.sdk.operatorframework.io/v2: {}
+  scorecard.sdk.operatorframework.io/v2: {}
+projectName: directpv-operator
+resources:
+- api:
+    crdVersion: v1
+    namespaced: true
+  domain: quay.io
+  group: charts
+  kind: DirectpvChart
```

Comment:
Could we style this all lowercase `directpvchart` or pretty name `DirectPVChart`?